test_setup.smtp.yml

```diff
@@ -0,0 +1,130 @@
+---
+# ========================================
+#  QE configuration of SMTP notifications
+# ========================================
+#
+# Overview of the configuration this playbook sets up:
+#
+# * postfix smtp server runs on usm_client machine, listening on public
+#   interface
+# * tendrl notifier is configured to send email via postfix smtp server running
+#   on usm_client directly (so mx entries are required for this to work)
```

Comment:
I think there should be: *...(so **no** MX entries are required for this to work)*.
Also I would use upper case for *MX*.